src/ucp/core/ucp_context.c

```diff
@@ -27,6 +27,16 @@
 
 #define UCP_RSC_CONFIG_ALL    "all"
 
+/* Seconds between keepalive rounds */
+#define UCP_CONTEXT_KEEPALIVE_TIMEOUT 1
```

Comment:
UCP_CONTEXT_DEFAULT_KEEPALIVE_TIMEOUT

Reply:
renamed to UCP_CONTEXT_DEFAULT_KEEPALIVE_TIMEOUT